README.md

```diff
@@ -19,22 +19,26 @@
 #
 -->
 
+<div align="center">
```

Comment:
I don't like very much the idea of changing the alignment of the logo, and also it won't be the same as for mynewt-core in this PR is merged.

Reply:
All right, dealigned.